src/action.ts

```diff
@@ -46,7 +45,7 @@ export default async function run() {
       }
     }
 
-    if (latestStage.status === 'failure') {
+    if (latestStage.status === 'failed' || latestStage.status === 'failure') {
```

Comment:
@arddluma Could we resolve if it's supposed to be `failed` or `failure`, the typing is used several times. Your type definition has `status` limited to `'idle'|'active'|'success'|'failed'` but you also changed this line from `failed` to `failure` in https://github.com/arddluma/cloudflare-pages-slack-notification/commit/7368aaecd05e4b09c20b5f30d9be41c1265dd72f .
I'm currently working on some typing fixes for error handling and `ghDeployment` as using `any` isn't good practice, but I'll contribute that when this PR is merged into master. (and hopefully when `failed` vs `failure` is resolved)
For some context: I'm working on a fork of this repo to add message customisation, so I thought I'd include some of the fixes once they're done.